content/news/core-monthly/2023/core-monthly-2022-12-01-2022-12-31.md

```diff
@@ -0,0 +1,377 @@
+
+**Update from Vue 2 to Vue 3**
+
+Thanks to [@NeOMakinG](https://github.com/NeOMakinG), core components written in Vue 2 are now fully migrated to Vue 3. You can see [this massive pull request](https://github.com/PrestaShop/PrestaShop/pull/28463) to see the migration details. Kudos to [@NeOMakinG](https://github.com/NeOMakinG) for amazing work!
+
+**Making PrestaShop core better**
+
+In every release, project members try to improve the software's core. This time I present you a pull request where [@FabienPapet](https://github.com/FabienPapet) migrated `Tools::str2url` so it uses the `StringModifier` utility class. Thanks to the migration, the function is now fully unit-tested. With such modifications, the project becomes more stable and reliable. You can see all the [implementation details in the pull request](https://github.com/PrestaShop/PrestaShop/pull/30357).
+
+I would also like to thank developers from [@StoreCommanderDev](https://github.com/StoreCommanderDev) for [fixing](https://github.com/PrestaShop/PrestaShop/pull/30540) a very tricky issue with the `Product::getAttributesResume`, that caused this method to be inconsistent. The bug they reported explains that this method could cause returning incorrect data in some scenarios.
+
+**Faceted search module**
+
+There's a new version of the Faceted Search module available. This release brings a set of new features and improvements, such as [setting a default template for filters](https://github.com/PrestaShop/ps_facetedsearch/pull/728), [performance improvements](https://github.com/PrestaShop/ps_facetedsearch/pull/727), and [changes](https://github.com/PrestaShop/ps_facetedsearch/pull/726) to simplify the module's codebase.
+
+Thanks to [@Hlavtox](https://github.com/Hlavtox), [@leemyongpakvn](https://github.com/leemyongpakvn), and [@zuk3975](https://github.com/zuk3975) for the work on this new version.
+
+**Changes in the productcomments module**
+
+Mickaël Fernandez[@micka-fdz]() reported that the `productcomments` module is causing some troubles with generated microdata on product listings. Not only Mickaël submitted a bug report, but he managed to [fix the issue](https://github.com/PrestaShop/productcomments/pull/150). The second interesting change from this module is coming from me. I [fixed](https://github.com/PrestaShop/productcomments/pull/149) some issues with the customer's last name anonymization. There was a critical error if the anonymization was enabled and the customer used the last name starting with a special character like e.g., `Ł`. This would cause comments not to be displayed. These two changes are coming to `productcomments` v5.0.3, which should be released in the upcoming weeks.
```

Comment:
```suggestion
Mickaël Fernandez[@micka-fdz]() reported that the `productcomments` module is causing some troubles with generated microdata on product listings. Not only Mickaël submitted a bug report, but he managed to [fix the issue](https://github.com/PrestaShop/productcomments/pull/150). The second interesting change from this module is coming from me. I [fixed](https://github.com/PrestaShop/productcomments/pull/149) some issues with the customer's last name anonymization. There was a critical error if the anonymization was enabled and the customer used the last name starting with a special character like e.g., `Ł`. This would cause comments not to be displayed. Both changes are coming to `productcomments` v5.0.3, which should be released in the upcoming weeks.
```